### PR TITLE
[CI] Format code

### PR DESCRIPTION
**Pull request by SaschaMann**

### CODE REVIEW

File: languages/go/exercises/concept/booleans/.docs/introduction.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ var a bool
 Go supports three logical operators that can evaluate expressions down to Boolean values, returning either `true` or `false`.
 
 | Operator   | What it means                              |
-| ---------- | ------------------------------------------ | ------ | --------------------------------------------- |
+| ---------- | ------------------------------------------ |
 | `&&` (and) | It is true if both statements are true.    |
 | `          |                                            | ` (or) | It is true if at least one statement is true. |
```
Comment (SaschaMann) [inline languages/go/exercises/concept/booleans/.docs/introduction.md:15]: I think this is wrong.

Comment (SleeplessByte) [reply languages/go/exercises/concept/booleans/.docs/introduction.md:15]: I think that was already wrong in their doc. It looked misformatted to me when they merged it.

Comment (SaschaMann) [reply languages/go/exercises/concept/booleans/.docs/introduction.md:15]: let's merge it then?

**Pull request merged**